torch/autograd/forward_ad.py

```diff
@@ -39,7 +40,8 @@ def enter_dual_level():
 
 
 def exit_dual_level(*, level=None):
-    r"""Function that can be used to exit a forward grad level.
+    r"""Develop a function to exit a forward grad level.
```

Comment:
Similar to `enter_dual_level`. Tell what the function does, not "Develop a function".

Reply:
Ok